apps/explorer/src/pages/epochs/EpochTimer.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useEpochProgress } from './useEpochProgress';
+
+import { useGetSystemObject } from '~/hooks/useGetObject';
+import { ProgressCircle } from '~/ui/ProgressCircle';
+import { Text } from '~/ui/Text';
+
+export function EpochTimer() {
+    // todo: replace this call when we have an endpoint for querying current epoch
+    const { data } = useGetSystemObject();
+
+    const { progress, label } = useEpochProgress(
+        data?.epochStartTimestampMs,
```

Comment:
Yeah I think it'd be cool if this hook just fetched the state object and did this itself.